unit/solvers/smt2_incremental/smt_to_string.cpp

```diff
@@ -0,0 +1,12 @@
+// Author: Diffblue Ltd.
+
+#include <testing-utils/use_catch.h>
+
+#include <solvers/smt2_incremental/smt_sorts.h>
+#include <solvers/smt2_incremental/smt_to_string.h>
+
+TEST_CASE("Test smt_sortt to string conversion", "[core][smt2_incremental]")
+{
+  CHECK(smt_to_string(smt_bool_sortt{}) == "Bool");
```

Comment:
👍🏻 I really like how clean these are because of the visitor - I feel like it was a good design choice.